### ISSUE TITLE
Add _VERSION_ for _BOARD_t

### ISSUE THREAD

**Issue by btrrj** [hack-technicolor/hack-technicolor#170]
**General info**

My gateway is currently running firmware version 17.1.7882-0001041 from TIM BRASIL (LIVE TIM)

- Product: DGA4130
- ISP: TIM BRASIL (LIVE TIM)
- Countries: BRAZIL, RIO DE JANEIRO
- Commercial name from the ISP: TIM
- Board: VBNT-K

**Did anybody ever manage to get root on that device already?**
I Tried almost all Type 2 root access, no success...

*Select all applicable*

- [ ] Yes, by using a root strategy listed in the wiki
- [ ] Yes, following another different strategy
- [x] No, as far as I know it has never been rooted

*Say something more about the adopted strategy here*

**Firmware versions**

*Please fill as many available info about each firmware versions you have ever heard about for this board. Leave unknown parts empty.*

- Version: 17.1
- Full version: 17.1.7882-0001041
- Custom firmware version strings used by ISP in addition to the above: ?
- RBI file name: ?
- RBI official download URL: ?
  - [ ] The link is restricted to ISP users or requires download password
- Other RBI download links for us to look into it: ?
- Raw bank dump download link: ?
- Serial console bootlog: ?
- Other potentially relevant info about this firmware version:

*repeat the same above block of info for each known version you heard about*

**Other details**

Hi all!

I´m from Brazil and i´m using a capped TIM DGA4130 modem here.

I want to root it (or install custom firmware) mainly for 2 things: 1- Use the USB Ports to share my printer and make an external HDD file server. 2- Have a fully functional router.
I tried to follow the type 2 root access, but did not work.

Here are the main gateway and information screen:

![dga4130](https://user-images.githubusercontent.com/6982046/124542303-89987f80-ddf9-11eb-8210-ff981b0d0d92.jpg)
![dga41302](https://user-images.githubusercontent.com/6982046/124542308-8a311600-ddf9-11eb-9144-7d46f42884a9.jpg)

**Comment by LuKePicci** [comment]
Type 2, strategy #A, did you try all Generic profiles from AFG?

**Comment by btrrj** [comment]
> Type 2, strategy #A, did you try all Generic profiles from AFG?

Yes, tried all of them, but always get access denied on root>root

**Comment by LuKePicci** [comment]
Ok, if you have a firmware upgrade option in the UI, follow Type 1 instructions. Pick the TIM Italy 1.0.3 from the repo as a Type 2 firmware when requested. Stop before bank planning, you will need to get a dump of the LIVE TIM Brazil firmware first.

**Comment by btrrj** [comment]
> Ok, if you have a firmware upgrade option in the UI, follow Type 1 instructions. Pick the TIM Italy 1.0.3 from the repo as a Type 2 firmware when requested. Stop before bank planning, you will need to get a dump of the LIVE TIM Brazil firmware first.
Ok, I will try here. How can i make the dump?

**Comment by btrrj** [comment]
> Ok, if you have a firmware upgrade option in the UI, follow Type 1 instructions. Pick the TIM Italy 1.0.3 from the repo as a Type 2 firmware when requested. Stop before bank planning, you will need to get a dump of the LIVE TIM Brazil firmware first.

Ok, i will try that. How can i make the Dump?

**Comment by LuKePicci** [comment]
There is a making raw dump guide in the wiki. Make sure to dump both firmware banks mtd3 and mtd4 (one contains the tim brazile firmware, the other contains the new firmware and it's useless but let's say we don't know which bank will be active, so dump both), dump the eripv2 mtd5 and mtd2 overlay as well. The contents of overlay are also available inside /overlay folder, if you manage to nake a tar.gz from there it would be awesome to give a look into it, there could be an URL to tim brazil RBI files, as well as your credentials you need to setup manually in case..

**Comment by btrrj** [comment]
> There is a making raw dump guide in the wiki. Make sure to dump both firmware banks mtd3 and mtd4 (one contains the tim brazile firmware, the other contains the new firmware and it's useless but let's say we don't know which bank will be active, so dump both), dump the eripv2 mtd5 and mtd2 overlay as well. The contents of overlay are also available inside /overlay folder, if you manage to nake a tar.gz from there it would be awesome to give a look into it, there could be an URL to tim brazil RBI files, as well as your credentials you need to setup manually in case..

Can you send me this raw dump guide link, please? I´m afraid to do it wrong haha

**Comment by LuKePicci** [comment]
yw ;-) https://hack-technicolor.readthedocs.io/en/stable/Resources/#making-dumps

**Comment by btrrj** [comment]
> yw ;-) https://hack-technicolor.readthedocs.io/en/stable/Resources/#making-dumps
> 
> PS: just for your awareness, upgrading via that "Firmware upgrade" option in the web UI is fine because these devices write the new firmware into the opposite bank when you do it via webui (which uses sysupgrade, more on this [here](https://hack-technicolor.readthedocs.io/en/stable/Resources/#different-methods-of-flashing-firmwares)), so you never overwrite the currently running firmware this way. Don't use TFTP because you don't know the actual status of your banks and this may result (50% chances) in your LIVE TIM firmware to be lost.

This was the guide i saw, it has to be on Linux to do, right?
Can you show me exactly which firmware i should try to use?

**Comment by btrrj** [comment]
I did a factory reset because Internet was down and now PPP is not connecting anymore. It shows more option on WebUI than before and now i can access them.

I need to reatores Internet now, how can I do it? In PPP I can't change any settings

**Comment by btrrj** [comment]
I managed to get root access, now I need the Internet back, please

**Comment by FrancYescO** [comment]
Unfortunately installing the GUI you made the chance to lose the LIVE firmware to 100% (bank planning was applied automatically).

Actually you need another device to get a dump of the LIVE firmware or some other way to get the RBI from LIVE.

**Comment by LuKePicci** [comment]
> I did a factory reset

That is correct, you always need a factory reste after changing firmware, I think the Type 1 guide also tells you to do so. It is perfectly normal to have no internet on the TIM Italy firmware, you will get internet again once we root the LIVE TIM one. This type 2 firmware from TIM Italy is only needed to get root first.

> I managed to get internet access back and root...

awesome!

> I installed Ansuel Gui, and it looks almost perfect!

Did you made the dumps I told you immediatelly after getting root? otherwise the LIVE TIM firmware is lost, you'll need to get a new device running LIVE TIM firmware.

**Comment by btrrj** [comment]
> Unfortunately installing the GUI you made the chance to lose the LIVE firmware to 100% (bank planning was applied automatically).
> 
> Actually you need another device to get a dump of the LIVE firmware or some other way to get the RBI from LIVE.

Damn, did not know that. I had no internet and getting angry with it... and bare 1.0.3 AGTEF was not letting me change PPP settings, so i tried the GUI. Only using it i was able to configure internet settings again.

Well, i will try with someone dump Live Tim firmware somehow someday.

Should i remain on 1.0.3 firmware with GUI or i can update to something newer, like 2.2.0 (is it the latest?) is it recommended? What is the best thing to do now that i have a rooted device to optimize?

I couldn't yet manage to configure VOIP settings. Don´t know which ones i should use.

**Comment by FrancYescO** [comment]
You can update to 2.2.1, it's better.

If your ISP does not provide VoIP parameters (also user and password) you will be not able to configure voip, the only way was to stole parameters from the previous firmware (or from a new modem with LIVE firmware if it get autoconfigured when connected to your line)

**Comment by btrrj** [comment]
> You can update to 2.2.1, it's better.

Can you give me the link to download 2.2.1?
Best way is update it directly from Ansuel Gui right?

**Comment by LuKePicci** [comment]
We are waiting for someone on TIM Brazil who manage to get root on his VBNT-K without wiping its current firmware so we can  get a dump or and RBI for it.
